docs/openlaw-elements/README.md

```diff
@@ -145,14 +145,36 @@ variables={variables}
 
 In addition to the required parameters, we offer support for the following additional parameters.
 
-### sections
+### inputProps
 
-If you have organized the template variables into custom sections (overriding how they are organized in the template source itself), you may pass these as a parameter for the form to use.
+Pass your own props to the underlying `input`, `select`, `textarea` components.
+
+- **Pass to all** `'*': { ... }`: This will spread props to _all_ underlying user input components. If specific OpenLaw element types are present, they will override any identical props spread in from `*`.
+- **Pass to type** `Address: { ... }`: This will spread props to specific _types_ of underlying user input components. If the wildcard (`*`) key is present, it will spread in its props, but not override any of the type's props.
+
+**Available keys:**
+
+- `*`
+- `Address`
+- `Choice`
+- `Date`
+- `Identity`
+- `Image`
+- `LargeText`
+- `Number`
+- `Text`
+- `YesNo`
 
 ```
-sections: Array<any>
+inputProps: {
+ [string]: any
+} => {[string]: any}
```

Comment:
Just wanted to check this type example with you before merging - I was getting some flow errors in `openlaw-app` with this. But when changed to the below, it seemed ok:
```
inputProps: {[string]: {[string]: any}}
```

Reply:
Thank you for catching that. Your version is correct.

Reply:
Ok thanks for confirming. I can update that and get this merged.